src/github.com/matrix-org/dendrite/roomserver/storage/previous_events_table.go

```diff
@@ -0,0 +1,72 @@
+package storage
+
+import (
+	"database/sql"
+	"github.com/matrix-org/dendrite/roomserver/types"
+)
+
+const previousEventSchema = `
+-- The previous events table stores the event_ids referenced by the events
+-- stored in the events table.
+-- This is used to tell if a new event is already referenced by an event we 
```

Comment:
More fun incomplete sentences.

Basically, this table stores all the `prev_events` event IDs for every event ever seen ever, right?